### PR TITLE
Replicated first screen of the targeted app

### PR DESCRIPTION
**Pull request by xor3r**
**What was implemented** 
- [x] Dropdown menus
- [x] Containers with paddings
- [x] AppBar with search
- [x] Bottom bar with menu
- [x] ListView with horizontal scrolling
- [x] Responsive design while rotating screen

**What needs improvement**
- [ ]  Some containers need to use dynamic placing

### CODE REVIEW

**Comment by xor3r** [comment]
**Vertical preview**

**Comment by xor3r** [comment]
**Horizontal preview**

**Comment by xor3r** [comment]
**Original app for reference**

**Review by Denys-Doskochynskyi — APPROVED** [review]
Really 'a lot' of nice code, perfect copying from the original application, pls merge :)
 (just one note, maybe if you try to create your own widget for the same item and use map or builder, it greatly optimizes your work)

**Review by redd4ford — APPROVED** [review]
MediaQuery approach gang's here :)
That's a lot of work!
You might want to dynamically resize the text depending on the screen size, but there's no need to overengineer the app as constant size works perfectly fine here. I really loved your app!

**Comment by user-nazar** [comment]
This is a very good work. Improve this application in the future

File: lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,876 @@
...
+                                value: dropdownCurrency,
+                                underline: const SizedBox(),
+                                alignment: Alignment.center,
+                                icon: const Icon(Icons.arrow_drop_down),
+                                iconSize: 20,
+                                elevation: 8,
+                                style: const TextStyle(color: Colors.white),
+                                onChanged: (String? newValue) {
+                                  setState(() {
+                                    dropdownCurrency = newValue!;
```
Comment (igor-leshkevych) [inline lib/main.dart:189]: extra work, wasn't needed for this lab

File: lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,876 @@
...
+                          style: TextStyle(
+                              fontSize: 18,
+                              color: _scrollableRowData[index]['isSelected'] ==
+                                      true
+                                  ? Colors.blue
+                                  : Colors.white),
+                        ),
+                        onTap: () {
+                          setState(() {
+                            _scrollableRowData
```
Comment (igor-leshkevych) [inline lib/main.dart:533]: extra work, wasn't needed for this lab

**Review by igor-leshkevych — APPROVED** [review]
@damoklov 
Your `MyApp` widget is way too big, you need to split it into multiple (you can try extract separate sections or smaller sub-widgets). The rest of the app is nice, I appreciate your attention to details :)

+8/8

**Pull request merged**